README.md

```diff
@@ -95,6 +95,7 @@ Many variables have default values and are therefore optional.
 | `comment-prefix`              | `.`                   | The character which follows `//` to signify documentation to transcribe.      |
 | `opening-delimiter`           | `` ```javascript ``   | The opening delimiter of doctest blocks in the source files.                  |
 | `closing-delimiter`           | `` ``` ``             | The closing delimiter of doctest blocks in the source files.                  |
+| `module-type`                 | `commonjs`            | The type of module doctest should use. One of `commonjs`, `amd`, or `esm`.    |
```

Comment:
Is it necessary to mention doctest here? This variable could also be used for other purposes.

Reply:
But it's not. I think it's better to change the description once it's used for other purposes.

Reply:
Okay. :)